Limitations.md

```diff
@@ -6,7 +6,6 @@
     * [Runtime commands](#runtime-commands)
         * [checkpoint and restore](#checkpoint-and-restore)
         * [events command](#events-command)
-        * [ps command](#ps-command)
```

Comment:
Ah, I see, as the `ps` section has gone already ... (I had to go look to see why we were removing it ;-) ).